Tournament_Calculator.py

```diff
@@ -0,0 +1,323 @@
+#import modules
+import openpyxl
+import random
+import re
+
+#Player Name/Hero dictionary and set variable for below function
+y=0
+Player_List = {}
+
+
+#Function to populate dictionary with player name/Hero
+def Player_Creation(Hero):
+    global y
+    for row in sheet_obj.iter_rows(1):
+        for cell in row:
+            if cell.value == Hero:
+                z = (sheet_obj.cell(row=(cell.row), column=(2)).value)
+                x = 0
+                if z == 0:
+                    pass
+                else:
+                    while x < int(z):
+                        New_Player = "Player" + str(y)
+                        Player_List[New_Player] = Hero
+                        y +=1
+                        x +=1
+                return y
+
+ 
+
+#Set Excel defaults for player input
+path = "Input Data.xlsx"

```

Comment:
Just a heads-up that paths can get a little weird if there are spaces in them, though moreso in linux than in windows. Either way, it's good practice not to have spaces in filenames that you will be providing to your code, use _ instead